bin/console.php

```diff
@@ -56,17 +60,24 @@ static function (int $errorCode, string $message = '', string $file = '', int $l
         $httpClient,
         $githubToken
     ));
-    $createReleaseText    = new CreateReleaseTextThroughChangelog(JwageGenerateChangelog::create(
+    $commit               = new CommitFileViaConsole();
+    $push                 = new PushViaConsole();
+    $releaseChangelog     = new ReleaseChangelogViaKeepAChangelog($commit, $push);
+    $createCommitText     = new CreateReleaseTextThroughChangelog(JwageGenerateChangelog::create(
         $makeRequests,
         $httpClient
     ));
-    $push                 = new PushViaConsole();
+    $createReleaseText    = new AppendingCreateReleaseTextAggregate([
```

Comment:
The name `AppendingCreateReleaseTextAggregate` is quite terrible - can we synthesise into a small sentence?

Reply:
How about `AppendSeveralReleaseTexts`?

Reply:
s/append/concatenate, but otherwise  :+1:

Reply:
Never mind - I like your suggestion later of `ConcatenateMultipleReleaseTexts`.